From 21063ef81aea2c026b7c65c33012d12328efe1d0 Mon Sep 17 00:00:00 2001
From: Sanjeev Gupta <ghane0@gmail.com>
Date: Tue, 29 Nov 2016 17:31:22 +0800
Subject: [PATCH 062/268] Remove broken internal link

Is there any info available on the crypto status codes,
similar to the KISS or FLASH codes?
---
 docs/decode.txt | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/docs/decode.txt b/docs/decode.txt
index 24d88cb..8d2f8ac 100644
--- a/docs/decode.txt
+++ b/docs/decode.txt
@@ -23,7 +23,9 @@ include::includes/install.txt[]
 * link:#clock[Clock Status Word]
 * link:#flash[Flash Status Word]
 * link:#kiss[Kiss Codes]
-* link:#crypto[Crypto Messages]
+// The crypto link goes nowhere.  TODO: locate this info and add here
+// -- Sanjeev 20161129
+// * link:#crypto[Crypto Messages]
 
 '''''
 
-- 
2.7.4

